OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -178,6 +178,7 @@ int main()
     {
         // Build the hopper.
         auto hopper = buildHopper();
+        hopper.finalizeFromProperties(); //required due to known bug
```

Comment:
We should also add `model.finalizeFromProperties()` to testREADME to be consistent.